src/Message/AbstractRequest.php

```diff
@@ -73,6 +83,10 @@ protected function xml2array($xml)
     {
         $arr = [];
 
+        if ($xml == false) {
```

Comment:
Maybe it is better to test for any invalid value here.

```suggestion
        if (! $xml) {
```

Reply:
Sounds good to me!